at-tanvo/D3-Exercise3.rb

```diff
@@ -0,0 +1,16 @@
+puts 'Nhap phan tu mang :'
+a = gets.to_i
+arr = []
+for i in 1..a
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/353f9504b403c43824ace9c0eeb9caf5548b0836/at-tanvo/D3-Exercise3.rb#L4"></span>Style/For: Prefer <code>each</code> over <code>for</code>. (https://github.com/rubocop-hq/ruby-style-guide#no-for-loops)</td>
    </tr>
  </tbody>
</table>